frontend/src/components/table/TableContextMenu.js

```diff
@@ -120,22 +124,45 @@ class TableContextMenu extends Component {
       },
 
       onComplete: () => {
-        const { references } = this.state;
+        let offset;
+        const { references, contextMenu } = this.state;
+        const { y: initialY } = contextMenu;
+        const mainPanel = document.querySelector('.panel-vertical-scroll');
```

Comment:
I'm not sure how I feel about mixing in vanilla js here instead of using ReactDOM. Isn't it against some react best practices ?

Reply:
I don't think that using the ref would help in here as you don't have any fnct exposed that would allow you.to close the calendars.